README.md

```diff
@@ -17,6 +17,16 @@ Software artifacts can be built using [Nix](https://nixos.org/nix/).
 To build the Haskell project, run `nix-build -A haskell`.
 To build the Coq project, run `nix-build -A coq`.
 
+To build the Coq part without Nix, you can try:
+
+1. download and extract VST from the URL in `vst.nix`
+1. in the VST directory, do: `make compcert/lib/Integers.vo sha/SHA256.vo sha/functional_prog.vo`
```

Comment:
I'd like to instead add a step here that says something like "export the VST directory into the COQPATH environment variable" and then remove all references to VST-2.3 from the `_CoqProject` and `dune` files.